### DISCUSSION TITLE
Is it possible to configure projects to run in Linux containers in the inner-loop when developing on Windows?

### DISCUSSION THREAD

**Discussion by dmi-harlow** [microsoft/aspire#6590] [Q&A]
I plan to deploy to Azure Container Apps which runs Linux containers, but my development environment is on Windows. Aspire container resources run locally in Linux containers in Docker, but project resources like ASP.NET Web API's run as Windows executables. My concern is that I might not find Windows vs. Linux issues until after deployment to the cloud. Is it possible to configure projects to run as Linux containers during local development on Windows?

BTW- I already tried adding Docker launch profiles to the Aspire projects but get the following error launching the Aspire AppHost project: "The project doesn't know how to run the profile with name 'Container (Dockerfile)' and command 'Docker'."

The workaround of course is to launch the web projects standalone to Docker in Linux containers and run component tests against that container, so I wouldn't say this is a stopper. Also, my concern mainly applies to third party libraries that claim to run on both Windows and Linux, so that is typically a one-time verification vs. something that needs to be tested continuously. If running the Aspire inner-loop with projects in Linux containers is not currently supported, I can accept that. I just didn't see anything in the docs about it one way or another, which is why I posted this as a question, not an issue.

**Comment by AldeRoberge** [comment]
I am wondering the same, I see this question as closed but no answer. Did you find a solution? Thanks!

**Reply by dmi-harlow** [reply]
I found that it is not supported. I closed the question after realizing it isn't really necessary. Whenever we have a concern about a third party library running in Linux (which is very rare, but happens), we test that library separately in Linux before adopting it in our Aspire projects.
